source/user-manual/reference/ossec-conf/localfile.rst

```diff
@@ -595,6 +597,96 @@ For example, we may want to read a Python Traceback output as one single log, re
       <multiline_regex replace="wspace">^Traceback</multiline_regex>
    </localfile>
 
+ignore
+^^^^^^^^^^^^^^^
+
+This allows you to configure a pcre2 expression to ignore specific log lines. Multiple expressions can be configured by repeating the ignore label. The log will be ignored as soon as one of the expressions matches, OR behavior.
```

Comment:
- Better using impersonal sentences (avoid using `This allows you`).
- pcre2 is uppercase
- Let's simplify the description
```suggestion
Specify a regular expression to ignore log lines or command outputs when matching. Whether several `ignore` labels are defined, entries are ignored when matching any of the specified ones.
```